replacer.go

```diff
@@ -66,6 +66,17 @@ func (r *Replacer) Get(variable string) (interface{}, bool) {
 	return nil, false
 }
 
+// GetString gets a value from the replacer. It returns
+// the value and whether the variable was known.
+func (r *Replacer) GetString(variable string) (string, bool) {
+	for _, mapFunc := range r.providers {
+		if val, ok := mapFunc(variable); ok {
+			return toString(val), true
+		}
+	}
+	return "", false
```

Comment:
Maybe just call `Get()` and then wrap the return value in `toString()`, as opposed to copying+pasting the same logic.